ml/src/main/scala/org/apache/spark/ml/h2o/algos/H2OAutoML.scala

```diff
@@ -193,7 +196,7 @@ trait H2OAutoMLParams extends DeprecatableParams {
     stoppingTolerance -> 0.001,
     stoppingMetric -> ScoreKeeper.StoppingMetric.AUTO,
     nfolds -> 5,
-    convertUnknownCategoricalLevelsToNa -> false,
+    convertUnknownCategoricalLevelsToNa -> true,
```

Comment:
Is this the default behavior in H2O-3?

Reply:
Default is false in H2O. For now, I will keep it consistent with current state of Sparkling Water, later we can iterate